shoes-core/lib/shoes/line.rb

```diff
@@ -19,6 +21,25 @@ def create_dimensions(point_a, point_b)
       style[:y2] = point_b.y
     end
 
+    def in_bounds?(x, y)
+      # c is (x, y)
+      # a is the left-most point
+      # b is the right-most point
+      a, b = [point_a, point_b].sort_by { |a, _| [a.x, a.y] }
```

Comment:
couldn't this be a comparison? Like point_a.x < point_.x ? And then assign values based on an if or a ternary?